test/setup-kind.sh

```diff
@@ -6,6 +6,7 @@ set -o pipefail
 
 IMAGE_TAG="${IMAGE_TAG:-edge}"
 NO_START_KIND="${NO_START_KIND:-}"
+CI_ENABLED=${CI_ENABLED:-}
```

Comment:
When `CI_ENABLED` is set, the script would install, otherwise upgrade.
a) can we rename this variable to make the intent more clear. e.g. `HELM_UPDATE` or similar.
b) shouldn't we install by default? So I would opt to be specific in case I want to update by setting `HELM_UPDATE=1`

Reply:
Oh, I guess you got this wrong, when `CI_ENABLED` is set, the script will use the chart from `charts/` directory but. when it is not set the script will fetch chart from `someengineering.github.io/helm-charts` artifactory via the published charts as is.
The helm upgrade & install are both same since we are using `-i` flag with helm upgrade which essentially means to install ifNotPresent.

Reply:
@aquamatthias ^^

Reply:
Ah I misunderstood indeed  - thanks. This is fine.